app.js

```diff
@@ -13,8 +13,15 @@ const init = () => {
   module.exports = app;
 
   // Client must send "Content-Type: application/json" header
-  app.use(bodyParser.json());
-  app.use(bodyParser.urlencoded({ extended: true }));
+  app.use(bodyParser.json({
+    parameterLimit: 10000,
+    limit: 1024 * 1024 * 10
```

Comment:
What are these magic numbers? Maybe we can use some default values and add the possibility to modify them with the config files. What do you think? @ilanusse